components/stepper/step-template.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Template
+page_title: Template
+description: Step Template for the Stepper for Blazor.
+slug: step-template
+tags: telerik,blazor,stepper,step,template
+published: True
+position: 23
+---
+
+
+# Step Template
+
+You can customize the whole appearance of the steps through their Template. It allows you to control the rendering and styles of both indicators and labels of the step.
+
+>caption Use the Step Template to customize the appearance of the steps. The result from the snippet below.
+
+![Step Template](images/step-template-example.png)
+
+````CSTHML
```

Comment:
fix snippet formatting and indent. See if you can avoid the `@<text>` tags to simplify the code.

Reply:
Snippet formatted. Unfortunately `@<text>` cannot be avoided as this allows to add markup in the code block